CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+* support isFileLockEnabled for buckets
```

Comment:
We used to start an item with a capital letter here, in SDK, and in CLI. Please follow that convention